test/test_pubsub.py

```diff
@@ -51,6 +53,13 @@ async def test_subscribe_multiple_channels(mock_pubsub):
     assert (1, 2, 3) == tuple(mock_pubsub._subscriptions.keys())
 
 
+@pytest.fixture()
+def mock_pubsub_subscriptions(mocker, mock_pubsub):
```

Comment:
Something to check. I looks that there is no need to inherit after `mocker here`

You could try to apply this change and see if it works.

```suggestion
def mock_pubsub_subscriptions(mock_pubsub):
```